[FIX] Update text to reflect that BackgroundSuppressionPulseTime is recommended (#1891)
bids-standard/bids-specification: src/appendices/arterial-spin-labeling.md

The `LabelingDuration` is defined as the total duration of the labeling pulse train in seconds.
`PostLabelingDelay` is the time in seconds after the end of the labeling until the middle of the excitation pulse applied
to the imaging slab (for 3D acquisition) or first slice (for 2D acquisition).

> Additionally, the `BackgroundSuppressionPulseTime`'s is recommended in the case where `BackgroundSuppression` was applied.

Reviewer: ```suggestion
Additionally, the `BackgroundSuppressionPulseTime` is RECOMMENDED if `BackgroundSuppression` was applied.
```
